lib/fog/openstack/requests/dns_v1/get_quota.rb

```diff
@@ -0,0 +1,26 @@
+module Fog
+  module DNS
+    class OpenStack
+      class V1
+        class Real
+          def get_quota(project_id)
+            request(
+              :expects => 200,
+              :method  => 'GET',
+              :path    => "quotas/#{project_id}"
+            )
+          end
+        end
+
+        class Mock
+          def get_quota(_project_id)
+            response = Excon::Response.new
+            response.status = 200
+            response.body = data[:quota_updated] || data[:quota]
+            response
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
The path of this file should be `lib/fog/dns/openstack/v1/requests/get_quota.rb`.

This follows the pattern we have defined right now in other providers. Could you move this file and other ones that do not follow this pattern please?